versioned_docs/version-8.7/components/concepts/document-handling.md

```diff
@@ -256,7 +258,15 @@ This feature will not work for public processes started by forms.
 
 ![document handling in tasklist](./assets/document-handling-tasklist.png)
 
-#### Outbound Connectors
+#### Start forms
```

Comment:
@marcosgvieira Who is best to work with on outlining the steps for start forms, user task forms, and displaying a document in a user task?

Reply:
Perhaps @volodymyr-melnykc, but want to double check.

Ideally it would be nice to have an engineer outline these steps (with a screenshot if necessary), and I can further polish and refine the content.